src/server.conf.in

```diff
@@ -10,7 +10,7 @@ listen = 127.0.0.1:8080
 ; HTTP socket write timeout in seconds.
 ;writetimeout = 10
 ; Use basePath if the server does not run on the root path (/) of your server.
-;basePath = /some/sub/path/
+basePath = /webrtc/
```

Comment:
OK. Can you make a change in the plugin then, which currently shows `/webrtc/` in both the config generator and the basepath in the advanced settings?